astropy/table/table.py

```diff
@@ -2855,6 +2858,7 @@ def from_pandas(cls, dataframe, index=False):
         --------
         Here we convert a :class:`pandas.DataFrame` instance to a `QTable`.
 
+          >>> import numpy as np
```

Comment:
Why this new import?

Reply:
It is more like a missing import. I think you are using numpy that is imported somewhere else (perhaps by coincidence), so your example was not strictly standalone.